index.html

```diff
@@ -92,113 +135,16 @@ <h2 class="sub-title-d">Project name goes here</h2>
                         <li>Ruby on Rails</li>
                         <li>Javascript</li>
                       </ul>
-                      <div class="btn-project">
+                      <!------<div class="btn-project">
                       <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
-                      </div> 
+                      </div> ----->
```

Comment:
- [ ] Please 🙏🏻 remove these line of code from your `file` because having commented code in the codebase makes it cluttered, confuses developers, and can create maintenance and version control issues. Kindly fix it and it should be avoided for clean, professional and standardized code.